CRM/Admin/Form/RelationshipType.php

```diff
@@ -97,7 +97,7 @@ public function getDefaultEntity() {
    * We do this from the constructor in order to do a translation.
    */
   public function setDeleteMessage() {
-    $this->deleteMessage = ts('WARNING: Deleting this option will result in the loss of all Relationship records of this type.') . ts('This may mean the loss of a substantial amount of data, and the action cannot be undone.') . ts('Do you want to continue?');
+    self::setEntityDeleteMessage(ts('Relationship'));
```

Comment:
@mlutfy can you comment on whether this interpolation is equally good from your point of view? I originally assumed that the way it was before (with a line for each entity) was 'better' but not so sure now...

Reply:
I started changing it separately for each entity but then realised that nearly every entity with a delete action shares exactly the same wording (except the entity name) so I think this will reduce the number of strings that need translating significantly.

Reply:
So - I'm not sure if it is clear without the world - I'm not inclined to make a change with a negative impact on usability for non-anglophones where there is such a minor code repetition decrease